src/xdocs/config_misc.xml

```diff
@@ -366,24 +366,24 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
         <source>
 0   switch(a) {
 1     case "1":
-2        int k = 7;
-3        // it is OK
+2       int k = 7;
+3       // it is OK
 4     case "2":
-5        int k = 7;
-6     // it is OK
+5       int k = 7;
+6       // it is OK
```

Comment:
This should be at the same position as case on next line.

Reply:
@rnveach Shouldn't there be an ident after the case for String "2"?
```
case "2":
  int k = 7;
  // it is OK
case "3":
  if (true) {}
```

Reply:
`int k` should be indented +2, but the original comment was at the same indentation as the case.
If you compare the original indentation for case 1 and 2, it is showing that the comment is acceptable at either the same level as the case, or at the same level as the `int k`. Your code as it is now is just showing the same example twice.

This specific line should be:
````
case "2":
  int k = 7;
// it is OK
````

Reply:
@rnveach Done